README.md

```diff
@@ -344,7 +344,7 @@ on the CLDR data's license.
 
 # COPYRIGHT AND LICENSE
 
-This software is copyright (c) 2014 by Fayland Lam.
+This software is copyright (c) 2018 by Fayland Lam.
```

Comment:
Ditto, and note to be careful about this unless upstream really does update; perhaps better to revert this particular hunk.

Reply:
OK, since this has been brought up before, I'll remove everything except for the version bump (but it was all due to `dist.ini`, not manual changes.